src/main/java/lavalink/client/io/jda/JdaLink.java

```diff
@@ -38,7 +38,7 @@ void connect(VoiceChannel channel, boolean checkChannel) {
         if (!channel.getGuild().equals(getJda().getGuildById(guild)))
             throw new IllegalArgumentException("The provided VoiceChannel is not a part of the Guild that this AudioManager handles." +
                     "Please provide a VoiceChannel from the proper Guild");
-        if (!channel.getGuild().isAvailable())
+        if (!channel.getJDA().isUnavailable(channel.getGuild().getIdLong()))
```

Comment:
negation must be removed here

Reply:
That is correct, I must have looked over that